### PR TITLE
Implement jest testing for pages

### PR DESCRIPTION
**Pull request by Otegaa**
### In this PR,
- I implemented jest testing for home, quotes, calculatorLogic and calculatorDisplay.

### CODE REVIEW

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Otegaa 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that requires you to work with your coding partner. please you guys should collaborate together and resubmit for a review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/testing/CalculatorLogic.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  it('should return a sum if "+" is assigned between two numbers', () => {
+    const result = operate(1, 5, '+');
+    expect(result).toEqual('6');
+  });
+
+  it('should return a multiplication if "x" is assigned between two numbers', () => {
+    const result = operate(10, 3, 'x');
+    expect(result).toEqual('30');
+  });
+});
```
Comment (julie-ify) [inline src/testing/CalculatorLogic.test.js]: - [ ] Kindly add more tests for all functionalities and scenarios in the calculate.js and operate.js files.

- [ ] **[OPTIONAL]** I suggest you separate the tests pertaining to each of the files into their own files

File: src/testing/Home.test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import '@testing-library/jest-dom';
```
Comment (julie-ify) [inline src/testing/Home.test.js:1]: - [ ] You have done well adding tests for simulating user actions using the testing library, but you are still missing snapshot tests for all your components. Kindly add snapshot tests for the components captured below.

  Pages folder | Components folder
  :---:|:---:
  rririw | duirir

File: src/testing/CalculatorDisplay.test.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    expect(getByText('-')).toBeInTheDocument();
+    expect(getByText('+')).toBeInTheDocument();
+    expect(getByText('.')).toBeInTheDocument();
+    expect(getByText('=')).toBeInTheDocument();
+  });
+});
+
+test('should show "0" by default', () => {
+  const { getByText } = render(<CalculatorDisplay />);
+  expect(getByText('0')).toBeInTheDocument();
```
Comment (julie-ify) [inline src/testing/CalculatorDisplay.test.js]: - [ ] One of your tests is failing. Kindly rectify it.

  image

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @Otegaa and @jkanyi-web,

Good job so far!

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/testing/CalculatorLogic.test.js (hunk 1 above)
Comment (Otegaa) [reply src/testing/CalculatorLogic.test.js:25]: @julie-ify , I’m CONFUSED, are you requesting for changes or approving us?

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi team 👋, 

Great job implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**